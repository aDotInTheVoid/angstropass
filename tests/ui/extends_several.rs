#[angstropass::langs]
mod langs {
    #[extends(foo bar)]
    mod baz {}
}

fn main() {}
