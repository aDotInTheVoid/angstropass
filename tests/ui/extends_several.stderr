error: unexpected token
 --> tests/ui/extends_several.rs:3:19
  |
3 |     #[extends(foo bar)]
  |                   ^^^
